### PR TITLE
Add stylelint-images plugin

### PR DESCRIPTION
**Pull request by ramasilveyra**
[stylelint-images](https://github.com/ramasilveyra/stylelint-images/)

**Suggestions about the introduction message are welcome!**

### CODE REVIEW

File: docs/user-guide/plugins.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ Plugins are rules and sets of rules built by the community that support methodol
 -   [`stylelint-scss`](https://github.com/kristerkari/stylelint-scss): Enforce a wide variety of SCSS-syntax specific linting rules (plugin pack).
 -   [`stylelint-selector-bem-pattern`](https://github.com/davidtheclark/stylelint-selector-bem-pattern): Specify a BEM pattern for selectors (incorporates [postcss-bem-linter](https://github.com/postcss/postcss-bem-linter)).
 -   [`stylelint-suitcss`](https://github.com/suitcss/stylelint-suitcss) - A collection of stylelint plugins for SUIT CSS, including deprecated `:root` rules (plugin pack).
+-   [`stylelint-images`](https://github.com/auth0/stylelint-images) - Check your CSS images to improve the performance and avoid common mistakes.
```
Comment (jeddy3) [inline docs/user-guide/plugins.md]: "Check your CSS images to improve the performance and avoid common mistake (plugin pack)."

**Comment by ramasilveyra** [comment]
@jeddy3 Thank you 😄 Changes ready!

**Comment by ramasilveyra** [comment]
@jeddy3 No problem, change ready! Also I added another commit to fix the same typo in another item https://github.com/stylelint/stylelint/pull/2487/commits/168cb364ee5b2da10f3b274ab17a49d28cd4b405

**Comment by jeddy3** [comment]
@ramasilveyra Thanks for making the amends :)

**Pull request merged**